sc-models/knowledge-base/metasystem_decomposition.scs

```diff
@@ -13,16 +13,37 @@ metasystem_decomposition
 
 @arc1 = (metasystem_decomposition -> rrel_1: knowledge_base_Metasystem);;
 
-@arc2 = (metasystem_decomposition -> .system_element_2290);;
+@arc2 = (metasystem_decomposition -> problem_solver_Metasystem);;
```

Comment:
If you want to use this system identifier, add it to the tex2scs-translator. system_element_2290 was formalised as a set of agents. You can add decomposition of it in Standard to fit system decomposition structure that is  required in sc-component-manager

Reply:
Same about subsystems